db/migrate/20250107164814_create_ite_client_records.rb

```
@@ -0,0 +1,20 @@
+class CreateIteClientRecords < ActiveRecord::Migration[7.0]
+  def change
+    create_table :ite_client_records do |t|
+      t.string :ite_client_id, null: false
+      t.date :date_of_birth
+      t.string :first_name
+      t.string :last_name
+      t.string :gender
+      t.string :full_name, null: false
+      t.string :middle_name
+      t.string :record_source, default: "claim_reconciled_claims", null: false
```

rubocop (1.69.1)

[convention] Prefer single-quoted strings when you don't need string interpolation or special symbols.

(Style/StringLiterals)